Felienne/hedy
# Modal changes & improvements

- Add timer to hide after X seconds the alert prompt: closes #684 and #677
- Add possibility to have default text in prompt: closes #728
- Add OK & cancel text and use them instead of "yes" and "no" in alert and prompt modals: closes #695
- Use modal to report that a program has been successfully saved

**How to test**
- Check that we now use the alert modal for 1) saving a program; 2) sharing a program. In those cases, check that the green box on the editor is no longer shown.
- Check that the alert modal disappears after 4 seconds after 1) saving a program; 2) sharing/unsharing a program; 3) copying a link to the clipboard.
- See that the button in the alert modal says "OK", not "Yes"
- See that the buttons in the prompt modal (when creating classes or changing an email in the admin) say "OK" and "Cancel" instead of "Yes" and "No"
- See that when changing an email in the admin, the prompt contains the existing email.
